src/EditorFeatures/CSharp/BlockCommentEditing/BlockCommentEditingCommandHandler.cs

```diff
@@ -170,7 +171,41 @@ private static bool IsCaretInsideBlockCommentSyntax(SnapshotPoint caretPosition)
             var trivia = syntaxTree.FindTriviaAndAdjustForEndOfFile(caretPosition, CancellationToken.None);
 
             var isBlockComment = trivia.IsKind(SyntaxKind.MultiLineCommentTrivia) || trivia.IsKind(SyntaxKind.MultiLineDocumentationCommentTrivia);
-            return isBlockComment && trivia.FullSpan.Start < caretPosition;
+            if (isBlockComment)
+            {
+                var span = trivia.FullSpan;
+                if (span.Start < caretPosition && caretPosition < span.End)
```

Comment:
What happens if we're in a `/**/` and we're between the `*/`? I'm not sure it's clear what _should_ happen, but it might be worth having a test for it.